ci-operator/config/stackrox/stackrox/config.py

```
@@ -761,12 +761,41 @@ def check_configs(data):
                 configs_ok = False
 
+        # 1. Check the rox-ci-image tag
+        # Check master/release branch configs do not use the "latest" floating
+        # tag for the rox-ci-image (apollo-ci). The "latest" tag is a moving
+        # target and should only be used for validation, not for CI jobs.
+        # Rules:
+        #   allow -stable, not -latest, for master
+        #   no -stable or -latest tags for release branches
+        # > latest is OK for in-flight PRs, but the check should still fail, to prevent merging such PR
+        if config.branch == 'master':
+            if 'latest' in tag:
```

Ruff (0.16.2)

[error] Undefined name `tag`

(F821)